Index: actuator-1.1.1/actuator.c
===================================================================
--- actuator-1.1.1/actuator.c
+++ actuator-1.1.1/actuator.c	2009-03-21 01:06:29.000000000 +0200
@@ -1356,7 +1356,7 @@
                curwidth-=colwidth;
                snprintf(buf, sizeof(buf),"%d%c ", menuvalue[itemindex], Pol);
                osd->DrawText(x+curwidth,y,buf,text,background,textfont,colwidth,rowheight,taRight);
-               snprintf(buf, sizeof(buf), tr(menucaption[itemindex]));
+               snprintf(buf, sizeof(buf), "%s", tr(menucaption[itemindex]));
                break;
              case MI_SYMBOLRATE:
              case MI_VPID:
@@ -1364,7 +1364,7 @@
                curwidth-=colwidth;
                snprintf(buf, sizeof(buf),"%d ", menuvalue[itemindex]);
                osd->DrawText(x+curwidth,y,buf,text,background,textfont,colwidth,rowheight,taRight);
-               snprintf(buf, sizeof(buf),tr(menucaption[itemindex]));
+               snprintf(buf, sizeof(buf), "%s", tr(menucaption[itemindex]));
                break;
              case MI_SATPOSITION:
                curwidth-=colwidth;
@@ -1382,7 +1382,7 @@
                curwidth-=colwidth;
                snprintf(buf, sizeof(buf),"(%d/%d)", transponderindex,menuvalue[MI_SCANSATELLITE]);
                osd->DrawText(x+curwidth,y,buf,text,background,textfont,colwidth,rowheight,taRight);
-               snprintf(buf, sizeof(buf),tr(menucaption[itemindex]));
+               snprintf(buf, sizeof(buf), "%s", tr(menucaption[itemindex]));
                break;
              default:    
                snprintf(buf,sizeof(buf),tr(menucaption[itemindex]),menuvalue[itemindex]);
