Index: actuator-1.1.1/module/actuator.c
===================================================================
--- actuator-1.1.1/module/actuator.c
+++ actuator-1.1.1/module/actuator.c	2009-07-15 04:35:29.000000000 +0300
@@ -293,7 +293,7 @@
     ioctl: actuator_ioctl,
 };
 
-void actuator_interrupt(int irq, void *dev_id, struct pt_regs *regs)
+void actuator_interrupt(void *private)
 {
 	write_lock(&lock);
 	if (stopping) {
@@ -373,7 +373,7 @@
      devfs_mk_cdev(MKDEV(major, 0), S_IFCHR | S_IRUSR | S_IWUSR, DRIVER_NAME);
 #endif     
      actuator_class = class_create(THIS_MODULE, DRIVER_NAME);
-     class_device_create(actuator_class,NULL,MKDEV(major,0),NULL,DRIVER_NAME);
+     device_create(actuator_class,NULL,MKDEV(major,0),NULL,DRIVER_NAME);
 
     /* set up timer function */
     init_timer(&timer);
@@ -418,7 +418,7 @@
 #ifdef CONFIG_DEVFS_FS
     devfs_remove(DRIVER_NAME);
 #endif    
-    class_device_destroy(actuator_class,MKDEV(major,0));
+    device_destroy(actuator_class,MKDEV(major,0));
     class_destroy(actuator_class);
     unregister_chrdev(major, DRIVER_NAME); 
 
